site/content/kapp-controller/docs/develop/authoring-command.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Authoring Command Reference
+---
+
+## Package
+
+`kctrl` authoring commands help users generate resources that interact with `kapp-controllers` packaging layer.
+
+### Initialising the package
+The `package init` command takes some user inputs and creates boilerplate for package creation.  It should be run before using the `release` command. 
+```bash
+$ kctrl package init
+```
+Supported flags:
+- `--chdir` _string_, Working directory with package-build and other config
+
+Note: We always suggest to run `pkg init` in an interactive flow.
+ 
+### Releasing the Package
+The `package release` command is used to generate Package and PackageMetadata resources.
+```bash
+$ kctrl package release -v v1.0.0 --repo-output packages
+```
+Supported flags:
```

Comment:
Is the `debug` flag supported? If yes, we have to add its documentation

Reply:
It is there as a part of global flags.